.codecov.yml

```diff
@@ -0,0 +1,29 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "20...100"
+
+  status:
+    project: no
+    patch: no
+    changes: no
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
+  layout: "reach,diff,flags,tree"
+  behavior: default
+  require_changes: no
+
+ignore:
```

Comment:
maybe we want to ignore some more file paths, for example:
* everything generated in ``restapi/``, ``client/``, ``models/`` etc. because not all code there will necessarily be used 
* ``subsystem/`` dir, which will never be called on unit-tests

Reply:
Yep, I can add them explicitly, but they are currently not reported because their packages do not contain test files.
You can see the report on codecov here: https://codecov.io/gh/openshift/assisted-service/tree/2bd6663b19374124d732df324b991910e3271bf5

Reply:
I have added few paths, let me know if others should be appended to the list

Reply:
No, I think it's a good list